src/pages/FilmView/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from "react";
+import { SearchEngine } from "../../components/SearchEngine";
+import { Film } from "../../components/Film";
+import axios from "axios";
+import { Menu } from "../../components/Menu";
+import "./FilmView.css";
+
+function FilmView() {
+  // fetch Data
+  const [films, setFilms] = useState([]);
+  // Filter toggle state
+  const [toggleFilter, setToggleFilter] = useState(false);
+
+  useEffect(async () => {
+    const response = await axios.get("https://ghibliapi.herokuapp.com/films");
+    setFilms(response.data);
+  }, []);
+  // Filter toggle function
+  const handleToggle = () => {
+    setToggleFilter(!toggleFilter);
+  };
+  return (
+    <>
+      <Menu />
```

Comment:
if you replace the menu with the button back would be great :D, I also don't know if there should have the search component

Reply:
Done. In my opinion we should keep there search component.